File: 백준/Bronze/34691. 대전과학고등학교를 사랑하십니까？/대전과학고등학교를 사랑하십니까？.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        while(true) {
            String word = scan.nextLine();
            if(word.equals("end")) break;
            if(word.equals("animal")) {
                System.out.println("Panthera tigris");
            } else if(word.equals("tree")) {
                System.out.println("Pinus densiflora");
            } else if(word.equals("flower")) {
                System.out.println("Forsythia koreana");
            }
        }
    }
}

File: 백준/Bronze/34691. 대전과학고등학교를 사랑하십니까？/대전과학고등학교를 사랑하십니까？.py
import sys
input=lambda:sys.stdin.readline().rstrip()
while True:
    word=input()
    if word=="end": break
    if word=="animal":
        print("Panthera tigris")
    elif word=="tree":
        print("Pinus densiflora")
    elif word=="flower":
        print("Forsythia koreana")